### DISCUSSION TITLE
Resolving real IP address out of a dyndns address?

### DISCUSSION THREAD

**Discussion by miquele** [daneren2005/Subsonic#1049] [Q&A]
hello,

i cant cant the app to work remotely whan I input my dyndns url (which works in the browser though). so I have to use the browser to tell me my ip address and then i copy the IP address into the server setting of the app and voila. it works.

this is really quite cumbersome.

why doesnt the app resolve the real IP address out of a dyndns address?

thanks,
miguel

**Comment by ignitionnight** [comment]
I use noip without issue and it shouldn't be too different from dyndns. Unless I'm mistaken, dsub doesn't resolve the real ip, dyndns should forward it. Are you adding the port to the end of your dyndns hostname? example my config in dsub is (username).no-ip.biz:4040 and it works just fine. If I leave out the 4040 I can't connect.

**Comment by daneren2005** [comment]
DSub uses the systems dns resolver so there isn't any reason a browser would be able to reach the site but not DSub.
